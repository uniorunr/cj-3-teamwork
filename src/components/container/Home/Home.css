.main-head {
  height: 100vh;
  display: flex;
  justify-content: center;
  align-items: center;
}

.main-head h2 {
  font-weight: bold;
  text-align: center;
  font-size: 4em;
  margin-bottom: 50px;
  color: #fff;
  text-transform: uppercase;
  text-shadow: 0 1px 2px #000, 0 2px 5px #000, 0 3px 10px #000;
}
