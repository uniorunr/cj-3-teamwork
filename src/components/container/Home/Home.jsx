import React, { Fragment } from 'react';
import PropTypes from 'prop-types';
import MainContent from './MainContent/MainContent';
import MainFooter from './MainFooter/MainFooter';
import './Home.css';

const Home = ({ teams, index }) => (
  <Fragment>
    <MainContent index={index} />
    <MainFooter teams={teams} />
  </Fragment>
);

Home.propTypes = {
  teams: PropTypes.arrayOf(
    PropTypes.shape({
      name: PropTypes.string,
      img: PropTypes.string,
    }),
  ),
  index: PropTypes.number.isRequired,
};

Home.defaultProps = {
  teams: 'teams',
};

export default Home;
