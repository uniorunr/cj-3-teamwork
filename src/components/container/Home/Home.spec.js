import React from 'react';
import renderer from 'react-test-renderer';
import Home from './Home';

it('renders correctly', () => {
  const tree = renderer.create(<Home teams={[{ name: 'a', img: 'img1' }, { name: 'b', img: 'img2' }]} />).toJSON();
  expect(tree).toMatchSnapshot();
});
