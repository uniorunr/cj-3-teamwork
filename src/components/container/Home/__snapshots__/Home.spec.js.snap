// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <div
    className="main-content"
  >
    <div
      className="main-director-text"
    >
      <h2>
        Что такое Lorem Ipsum?
      </h2>
      <p>
        Lorem Ipsum - это текст-"рыба", часто используемый в печати и вэб-дизайне. Lorem Ipsum является стандартной "рыбой" для текстов на латинице с начала XVI века. В то время некий безымянный печатник создал большую коллекцию размеров и форм шрифтов, используя Lorem Ipsum для распечатки образцов. Lorem Ipsum не только успешно пережил без заметных изменений пять веков, но и перешагнул в электронный дизайн. Его популяризации в новое время послужили публикация листов Letraset с образцами Lorem Ipsum в 60-х годах и, в более недавнее время, программы электронной вёрстки типа Aldus PageMaker, в шаблонах которых используется Lorem Ipsum.
      </p>
    </div>
    <div
      className="main-director-info"
    >
      <h2>
        Александровская Лариса Помпеевна
      </h2>
      <img
        alt="Александровская Лариса Помпеевна"
        src="https://upload.wikimedia.org/wikipedia/commons/0/0c/%C5%81arysa_Aleksandro%C5%ADskaja._%D0%9B%D0%B0%D1%80%D1%8B%D1%81%D0%B0_%D0%90%D0%BB%D0%B5%D0%BA%D1%81%D0%B0%D0%BD%D0%B4%D1%80%D0%BE%D1%9E%D1%81%D0%BA%D0%B0%D1%8F_%281937%29.jpg"
      />
      <p>
        Белорусская советская оперная певица (сопрано), режиссёр, публицист и общественный деятель. Народная артистка СССР (1940)
      </p>
    </div>
  </div>,
  <div
    className="main-footer"
  >
    <h2>
      Команда разработчиков
    </h2>
    <div
      className="developer-container"
    >
      <div
        className="developer-item"
      >
        <img
          alt="a"
          src="img1"
        />
        <p>
          a
        </p>
      </div>
      <div
        className="developer-item"
      >
        <img
          alt="b"
          src="img2"
        />
        <p>
          b
        </p>
      </div>
    </div>
  </div>,
]
`;
